app/src/client/components/form/TextInput.tsx

```diff
@@ -6,15 +6,17 @@ interface TextInputProps {
   value: string;
   placeholder: string;
   onChange: (value: string) => void;
+  isDisabled?: boolean;
 }
 
-export const TextInput: React.FC<TextInputProps> = ({ id, type, value, placeholder, onChange }) => (
+export const TextInput: React.FC<TextInputProps> = ({ id, type, value, placeholder, onChange, isDisabled }) => (
```

Comment:
we can use disabled here as well.

Reply:
Sure, make sense. I will push my changes shortly to the same PR